### PR TITLE
Cache file add module name

### PR DESCRIPTION
**Pull request by owen200008**
## What is the purpose of the change
1. support dubbo.application.enable-file-cache=false
2. cache file add application name

#9891 #9800

### CODE REVIEW

**Comment by AlbumenJ** [comment]
image

Please fix uts

File: dubbo-metadata/dubbo-metadata-api/src/main/java/org/apache/dubbo/metadata/AbstractServiceNameMapping.java [hunk 1]
```diff
@@ -60,7 +62,13 @@ public abstract class AbstractServiceNameMapping implements ServiceNameMapping,
 
     public AbstractServiceNameMapping(ApplicationModel applicationModel) {
         this.applicationModel = applicationModel;
-        this.mappingCacheManager = new MappingCacheManager("",
+        boolean fileCache = true;
+        Optional<ApplicationConfig> application = applicationModel.getApplicationConfigManager().getApplication();
+        if(application.isPresent()) {
+            fileCache = Boolean.TRUE.equals(application.get().getEnableFileCache()) ? true : false;
```
Comment (AlbumenJ) [inline dubbo-metadata/dubbo-metadata-api/src/main/java/org/apache/dubbo/metadata/AbstractServiceNameMapping.java:68]: replaced with `Boolean.parseBoolean`

File: dubbo-metadata/dubbo-metadata-api/src/main/java/org/apache/dubbo/metadata/MappingCacheManager.java [hunk 2]
```diff
@@ -36,7 +36,7 @@ public static MappingCacheManager getInstance(ScopeModel scopeModel) {
         return scopeModel.getBeanFactory().getOrRegisterBean(MappingCacheManager.class);
     }
 
-    public MappingCacheManager(String name, ScheduledExecutorService executorService) {
+    public MappingCacheManager(boolean fileCache, String name, ScheduledExecutorService executorService) {
```
Comment (AlbumenJ) [inline dubbo-metadata/dubbo-metadata-api/src/main/java/org/apache/dubbo/metadata/MappingCacheManager.java:39]: rename `fileCache` field to `enableFileCached`

**Review by AlbumenJ — CHANGES_REQUESTED** [review]
Add some uts for this pr

File: dubbo-metadata/dubbo-metadata-api/src/main/java/org/apache/dubbo/metadata/AbstractServiceNameMapping.java (hunk 1 above)
Comment (owen200008) [reply dubbo-metadata/dubbo-metadata-api/src/main/java/org/apache/dubbo/metadata/AbstractServiceNameMapping.java:68]: application.get().getEnableFileCache() type is Boolean，how to use Boolean.parseBoolean

File: dubbo-metadata/dubbo-metadata-api/src/main/java/org/apache/dubbo/metadata/MappingCacheManager.java (hunk 2 above)
Comment (AlbumenJ) [reply dubbo-metadata/dubbo-metadata-api/src/main/java/org/apache/dubbo/metadata/MappingCacheManager.java]: Sorry, I have made a spelling mistake. The proper filed name is `enableFileCache` which means whether enable file cache or not.

Comment (owen200008) [reply dubbo-metadata/dubbo-metadata-api/src/main/java/org/apache/dubbo/metadata/MappingCacheManager.java]: OK

**Review by AlbumenJ — APPROVED** [review]

**Pull request merged**